### PR TITLE
Gh 165 aws performance tests

### PR DESCRIPTION
**Pull request by ctas582**

### CODE REVIEW

File: deployment/aws/tests/performance-tests/ec2-userdata-scripts/run-gaffer-random-generator.sh [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+# Install required software
+yum remove -y java-1.7.0-openjdk
+yum install -y java-1.8.0-openjdk-devel
+
+MAVEN_VERSION=3.5.0
+MAVEN_DOWNLOAD_URL=http://www.mirrorservice.org/sites/ftp.apache.org/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz
+function install_dev_tools {
+	# Install git
+	if ! which git >/dev/null 2>&1; then
+		yum install -y git
```
Comment (p013570) [inline deployment/aws/tests/performance-tests/ec2-userdata-scripts/run-gaffer-random-generator.sh]: Some yum installs you do as the current user and some you do with sudo. Should they all be done with sudo?

File: deployment/aws/tests/performance-tests/ec2-userdata-scripts/run-gaffer-random-generator.sh [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+mkdir -p $INSTALL_DIR/lib
+cd $INSTALL_DIR
+
+tee -a etc/store.properties <<EOF
+gaffer.store.class=uk.gov.gchq.gaffer.accumulostore.AccumuloStore
+gaffer.store.properties.class=uk.gov.gchq.gaffer.accumulostore.AccumuloProperties
+
+accumulo.instance=$ACCUMULO_INSTANCE
+accumulo.zookeepers=$ZOOKEEPERS
+accumulo.table=$GAFFER_GRAPH_ID
```
Comment (p013570) [inline deployment/aws/tests/performance-tests/ec2-userdata-scripts/run-gaffer-random-generator.sh]: Can this property be remove? It is deprecated in Gaffer.

File: performance-testing/performance-testing-aws/src/main/java/uk/gov/gchq/gaffer/performancetesting/aws/CloudWatchMetricsListener.java [hunk 3]
```diff
@@ -112,6 +112,7 @@ public void update(final Metrics metrics) {
     @Override
     public void close() {
         this.cloudwatch.shutdown();
```
Comment (p013570) [inline performance-testing/performance-testing-aws/src/main/java/uk/gov/gchq/gaffer/performancetesting/aws/CloudWatchMetricsListener.java]: Is it possible for close to be called twice? If so we would need a null check around ```this.cloudwatch.shutdown();```

Similarly in a 'disconnect' method below.

File: deployment/aws/tests/performance-tests/ec2-userdata-scripts/run-gaffer-random-generator.sh (hunk 1 above)
Comment (ctas582) [reply deployment/aws/tests/performance-tests/ec2-userdata-scripts/run-gaffer-random-generator.sh]: Some scripts are run by a local user and some are run as root - which is the case here. But sure, I can run all yum commands via sudo to be consistent.

**Pull request merged**